.github/workflows/integration-tests.yml

```diff
@@ -42,7 +52,7 @@ jobs:
   
   stateless-validator:
```

Comment:
BTW, tried including `prove` in `action: [execute]` but it just takes hours. I had the sensation this wasn't that heavy, but actually tried in my own machine and CPU proving is really that slow.